app/views/find-help.nunjucks

```diff
@@ -4,7 +4,7 @@
 <h1 class="local-header--title--question">Find a pharmacy</h1>
 {% endblock %}
 
-{% block pageTitle %}Find a pharmacy{% if errorMessage %} - We can't find the postcode '{{location}}'{% endif %}{% endblock %}
+{% block pageTitle %}Find a pharmacy{% if errorMessage %} - We can't find the postcode '{{req_location}}'{% endif %}{% endblock %}
```

Comment:
Why is the title using the direct input and not the uppercased version as is used in the error message? I'd expect them to be the same.
I'd say having the title like this is correct but the message for inputs that are seen as postcodes but are invalid should also display the actual input rather than the uppercased input. What do you reckon?

Reply:
changed to show exact term entered.